projects/angular/tsconfig.lib.json

```
@@ -11,7 +11,8 @@
   "angularCompilerOptions": {
     "enableResourceInlining": true,
     "skipTemplateCodegen": true,
-    "strictMetadataEmit": true
+    "strictMetadataEmit": true,
+    "strictStandalone": false
```

```suggestion
    "strictMetadataEmit": true
```

This setting is not needed either.